javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithExtends.java

```diff
@@ -32,6 +32,10 @@
  */
 public interface NodeWithExtends<N extends Node> {
 
+    /**
+     * @return All extended types that have been explicitly added.
+     *   Note that this will not include {@code java.lang.Object} unless it is explicitly added (e.g. {@code class X extends Object {}})
```

Comment:
This likely needs to be double checked -- I recall being convinced at the time of writing this, but now I am less sure

Reply:
Yes, this is correct. 

This is supposed to be indicative of what is within the AST, thus refers to what is explicitly extended (within a class or interface).